src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/.template.config/dotnetcli.host.json

```diff
@@ -67,6 +67,22 @@
     "NoHttps": {
       "longName": "no-https",
       "shortName": ""
+    },
+    "RazorRuntimeCompilation": {
```

Comment:
This isn't used in this template why is it present in https://github.com/AzureAD/microsoft-identity-web/blob/33d7bacef9bcbecd29e955535bfbbdda20f73f41/ProjectTemplates/templates/BlazorServerWeb-CSharp/.template.config/dotnetcli.host.json#L71-L74?

Reply:
I'm not sure about `RazorRuntimeCompilation`, @JunTaoLuo 
I copied the templates from ASP.NET core last week ... maybe it was present before?

Reply:
I think it was present in 3.1 templates. I'll remove it.

Reply:
Yeah, this might be fine to remove here. We'd only enabled this feature in the MVC and Razor Pages templates